<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBaseWindow</class>
 <widget class="QMainWindow" name="DataBaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample DataBase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,4,4" columnstretch="2,1,0">
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Sample</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="8,1">
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_sumitSample">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_revertSample">
         <property name="text">
          <string>Revert</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_deleteSampleEntry">
         <property name="text">
          <string>Delete Entry</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_selectSample">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_addSampleEntry">
         <property name="text">
          <string>Add Entry</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="5">
        <widget class="QTableView" name="sampleView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Data</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,2,2,2,2,2" columnstretch="1,1,3,3,1">
       <item row="2" column="4">
        <widget class="QPushButton" name="pushButton_addDataEntry">
         <property name="text">
          <string>Add Entry</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="pushButton_revertData">
         <property name="text">
          <string>Revert</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="pushButton_submitData">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Recent Data on top</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="pushButton_deleteDataEntry">
         <property name="text">
          <string>Delete Entry</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pushButton_open">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="5" colspan="4">
        <widget class="QTableView" name="dataView"/>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox_showChemical">
         <property name="text">
          <string>Show Chemical</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_editHost">
      <property name="text">
       <string>Edit Host</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_editSolvent">
      <property name="text">
       <string>Edit Solvent</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Chemical</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,0">
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBox_chemical"/>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QLabel" name="label_Picture">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_ChemicalFormula"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_editChemical">
         <property name="text">
          <string>Edit Chemical</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
